tests/golden/control-plane/appcat/appcat/20_xrd_vshn_mariadb.yaml

```diff
@@ -5555,6 +5556,31 @@ spec:
                           description: Email necessary to send alerts via email
                           type: string
                       type: object
+                    network:
```

Comment:
why this came up here?

Reply:
MariaDB did not yet have the API to set a loadbalancer.